aws_terminate.sh

```diff
@@ -57,7 +57,10 @@ function aws_terminate() {
   elif [[ ${__ret} -ne 0 ]]; then
     return ${__ret}
   fi
-  aws_until_state terminated ${__id} ${__timeout} || return $?
+  if [[ ${DEBUG} ]]; then
```

Comment:
This will return True if the `$DEBUG` var is defined at all. You may want to do an explicit check of the value.

Reply:
Yeah, that's fine. I'm already doing that throughout the project.